scripts/calculator.js

```diff
@@ -0,0 +1,86 @@
+// CALCULATOR OBJECT
+const calculator = {
+  currentOperand: "",
+  previousOperand: "",
+  operator: null,
+}
+// REFRESH DISPLAY
+function updateDisplay() {
+  document.querySelector("[data-current-operand]").innerText = calculator.currentOperand;
+  document.querySelector("[data-previous-operand]").innerText = calculator.previousOperand;
+}
+//DETECT WHEN CLEAR BUTTON IS PUSHED
+document.querySelector("[data-clear]").addEventListener(
+  "click", clearCalculator)

```

Comment:
Not sure why this is being set dynamically - I think just do a standard `<button data-clear class="darkGrayButton" onclick="clearCalculator();">AC</button>`